scp/scpOperand.cpp

```diff
@@ -222,6 +224,10 @@ void SCPOperand::resolveModifiers()
       continue;
     }
 
+    auto const & it = Keynodes::m_deprecatedKeynodes.find(modifier);
+    if (it != Keynodes::m_deprecatedKeynodes.cend())
+      SCP_LOG_WARNING("Role relation `" << Keynodes::m_deprecatedKeynodeIdentifiers[modifier] << "` is deprecated. Use role relation `" << std::string(it->second) << "` instead.");
+
```

Comment:
questionable that two maps are needed for that
modifier may be in map number one and not in map number two